modules/olmv1-installing-an-operator.adoc

```diff
@@ -17,9 +17,8 @@ include::snippets/olmv1-known-issue-private-registries.adoc[]
 
 .Prerequisites
 
-* You have added a catalog to your cluster.
-* You have downloaded a local copy of the catalog file.
 * You have installed the `jq` CLI tool.
+* You have installed the `opm` CLI tool.
 * You have created a service account and assigned enough role-based access controls (RBAC) to install, update, and manage the extension you want to install. For more information, see _Creating a service account_.
```

Comment:
```suggestion
* You have created a service account and assigned enough role-based access controls (RBAC) to install, update, and manage the extension you want to install. For more information, see "Creating a service account to manage cluster extensions".
```
Minor change to align with the [OCP doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences) and to match the current title of the referenced docs.